lib/rt_utils.pm

```diff
@@ -0,0 +1,35 @@
+package rt_utils;
+use base "opensusebasetest";
+use testapi;
+use strict;
+use Exporter;
+
+our @EXPORT = qw(
+  select_kernel
+);
+
+sub select_kernel {
```

Comment:
We have `boot_grub_item` in `lib/bootloader_setup` - maybe it can be reused..seems very similar

Reply:
Yup, it works well! ```$self->boot_grub_item(2,3);```
http://localhost/tests/11373#step/boot_rt_kernel/7